pkg/kubelet/kubelet.go

```diff
@@ -823,6 +826,11 @@ type Kubelet struct {
 
 	// the number of allowed pods per core
 	podsPerCore int
+
+	// enableControllerAttachDetach indicates the Attach/Detach controller
+	// should manage attachment/detachment of volumes scheduled to this node,
+	// and disable kubelet from executing any attach/detach operations
```

Comment:
Discussed offline. Static pods created by kubelet create "mirror pod objects" in the API server, and those should get processed exactly the same way.